pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/PersistentTopicsBase.java

```diff
@@ -4398,8 +4397,11 @@ private CompletableFuture<Void> updatePartitionedTopic(TopicName topicName, int
      *
      * @param topicName     : topic-name: persistent://prop/cluster/ns/topic
      * @param numPartitions : number partitions for the topics
+     * @param ignoreConflictException : true: ignore ConflictException: subscription already exists for topic
```

Comment:
How about `* @param ignoreConflictException : If true,  ignore ConflictException`, too many ':'  is a little not clear~

Reply:
Thx. Make it better.